### PR TITLE
Add selection page styling

### PR DESCRIPTION
**Pull request by HiruNya**
image

### CODE REVIEW

File: src/pages/SelectionPage.js [hunk 1]
```diff
@@ -53,17 +55,28 @@ export default function SelectionPage({ changeView, setSelectedLocation }) {
...
   );
 }
 
 const selectionButton = ({ name, image, setSelectedLocation }) => (
   <button
     key={name}
-    className={
-      'flex button w-40 h-40 radius-lg border-2 drop-shadow-sm drop-shadow-md grid justify-center content-center'
-    }
+    className={'grid w-40 h-40 rounded-lg drop-shadow-xl justify-center content-center'}
```
Comment (beverleysun) [inline src/pages/SelectionPage.js]: nit: doesn't need to be in curly braces (also present in other places too :D)

**Review by f-fromont — CHANGES_REQUESTED** [review]
Please rename the blob to be "...BlobSvg"

**Review by beverleysun — CHANGES_REQUESTED** [review]
Change it bish

**Review by DarkMatterMatt — CHANGES_REQUESTED** [review]
Please improve before requesting another review.

**Review by DarkMatterMatt — APPROVED** [review]

**Review by beverleysun — APPROVED** [review]
:)

**Review by f-fromont — APPROVED** [review]
Nice

**Pull request merged**